### DISCUSSION TITLE
App Integrity call back triggered when using iphone real device

### DISCUSSION THREAD

**Discussion by deotamaaa** [talsec/Free-RASP-Flutter#146] [Q&A]
dHello guys, kindly confused with how it works

I already added the bundle Id and the team id matches with the app project, but when running it on real devices, the app Integrity callback is triggered. I'm implementing quit the app if it happens, now i can't debug with real devices, any explanation here? 

Thanks in advance!

**Comment by tompsota** [comment]
Hello @deotamaaa,

can you send us your bundle id, team id and watcher email to support@talsec.app, so that we can look at what triggered the kill exactly? Thank you.

Regards,
Tomas from Talsec
